.swiftlint.yml

```diff
@@ -11,6 +11,7 @@ disabled_rules:
 excluded:
   - AuroraEditorModules/.build # Where Swift Package Manager checks out dependency sources
   - AuroraEditor/Base/AuroraEditorTextView/STTextView # External package which needs to be 'fixed'
+  - AuroraEditor/Services/Model/GitHub
```

Comment:
This still needs to be fixed. We should create an issue so we don’t forget it.

Reply:
I'll remember, it's still a wip